# HG changeset patch
# User John Donoghue
# Date 1457645378 18000
#      Thu Mar 10 16:29:38 2016 -0500
# Node ID c5f319d0b7f57003e62ef00d65f0ac067a8b93e7
# Parent  f815b0145d1efe58c9195bb6881b6e9970255914
* configure.ac: remove spaces from LIBEXT assign

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2294,17 +2294,17 @@ AC_ARG_ENABLE([link-all-dependencies],
      yes) link_all_deps=yes ;;
      no)  link_all_deps=no ;;
      *) AC_MSG_ERROR([bad value $enableval for --enable-link-all-depenencies])
      ;;
    esac],
   [link_all_deps=no])
 AM_CONDITIONAL([AMCOND_LINK_ALL_DEPS], [test $link_all_deps = yes])
 
-LIBEXT = a
+LIBEXT=a
 AC_SUBST(LIBEXT)
 
 ## Dynamic linking is now enabled only if we are building shared
 ## libs and some API for dynamic linking has been detected.
 
 ## FIXME: A lot of the following duplicates the functionality of
 ## code generated by the dlopen option for LT_INIT.
 
